scripts/bootstrap-pallets/interfaces/assets/index.ts

```diff
@@ -0,0 +1,4 @@
+// Auto-generated via `yarn polkadot-types-from-defs`, do not edit
+/* eslint-disable */
+
+export * from "./types";
```

Comment:
Same, can we use composable.js?

Reply:
yeah i would be moving this to composable.js folder once that PR is merged